### DISCUSSION TITLE
What is the best method to reduce image jpg/png file size?

### DISCUSSION THREAD

**Discussion by nissansz** [python-pillow/Pillow#8278] [Q&A] [answered]
What is the best method to reduce image jpg/png file size? I saw Acrobat can reduce file size by 70% without quality loss, don't know how it was achieved

**Comment by radarhere** [comment, marked as answer]
When [saving JPEGs](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#jpeg-saving), there is an 'optimize' parameter, and when [saving PNGs](https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html#png-saving), there is 'optimize' and 'compress_level'. These will help you reduce file size.
